src/playready.js

```diff
@@ -0,0 +1,45 @@
+import videojs from 'video.js';
+import window from 'global/window';
+
+export const getMessageContents = (message) => {
```

Comment:
Could we get function level comments here and in the function below?  Just function description, params and return values?